contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -1071,7 +1071,7 @@ LANGUAGE 'pltsql';
 GRANT EXECUTE ON PROCEDURE sys.sp_tables TO PUBLIC;
 
 CREATE FUNCTION sys.fn_mapped_system_error_list ()
-returns table (sql_error_code int)
+returns table (pg_sql_state sys.nvarchar(5), error_message sys.nvarchar(4000), error_msg_keywords sys.nvarchar(4000), sql_error_code int)
```

Comment:
instead of keywords, we should use parameters

Reply:
Done!